### PR TITLE
Save for later

### PR DESCRIPTION
**Pull request by leanne1**

### CODE REVIEW

File: client/main.scss [hunk 1]
```diff
@@ -1,5 +1,5 @@
 $o-grid-mode: snappy;
-//$o-grid-debug-mode: true;
+$o-grid-debug-mode: true;
```
Comment (matthew-andrews) [inline client/main.scss]: The phantom debug mode

File: views/layout_2-improved.html [hunk 2]
```diff
@@ -48,7 +52,7 @@ <h3 class="article__title">{{{article.titleHTML}}}</h3>
 			<div data-o-grid-colspan="0 XL2">&nbsp;</div>
 			<div data-o-grid-colspan="12 L8 XL7">
 				{{#if article.byline}}
-					<cite class="article__byline">{{article.byline}}</cite>
+					<cite class="article__byline">By {{article.byline}}</cite>
```
Comment (matthew-andrews) [inline views/layout_2-improved.html]: Has George asked for this?

Comment (leanne1) [inline views/layout_2-improved.html]: Yep

File: client/main.scss (hunk 1 above)
Comment (leanne1) [inline client/main.scss]: :ghost:

**Pull request merged**